samples/bluetooth/peripheral_rscs/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: BLE RSCS sample
```

Comment:
Do not use BLE. Instead, use Bluetooth LE. See https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-B